keras_eval/eval.py

```diff
@@ -351,18 +368,66 @@ def get_image_paths_by_prediction(self, probs, labels=None, class_names=None, im
         return dict_image_paths_class
 
     def plot_images(self, image_paths, n_imgs=None, title=''):
+        # Works better defining a number of images between 5 and 30 at a time
         '''
 
         Args:
-            image_paths: list with image_paths
-            n_imgs: number of images to show
-            title: title for the plot
+            image_paths: List with image_paths
+            n_imgs: Number of images to show
+            title: Title for the plot
 
-        Returns:
+        Returns: Plots images in the screen
 
         '''
         image_paths = np.array(image_paths)
         if n_imgs is None:
             n_imgs = image_paths.shape[0]
 
         visualizer.plot_images(image_paths, n_imgs, title)
+
+    def compute_confidence_prediction_distribution(self, probs=None, combination_mode=None, verbose=1):
+        '''
+        Compute the mean value of the probability assigned to predictions, or how confident is the classifier
+        Args:
+            probs: Probabilities given by the model
+            combination_mode: Ways of combining the model's probabilities to obtain the final prediction.
+            verbose: Show text
+
+        Returns:
+
+        '''
+        if probs is None and self.probs is not None:
+            probs = self.probs
```

Comment:
probs = probs or self.probs fails when both are not None ...
I wanted to allow quick functions using the precomputed probs and at the same time allow to use the functions with other custom probs. 
Not sure if that is the best though.

Reply:
@snormore

Reply:
Those 2 lines should be equivalent to
```python
probs = probs or self.probs
```
since if we play out the different cases:
 1. If `probs is None` and `self.probs is None` then `probs` will be `None` with both approaches
 2. If `probs not None` and `self.probs is None (or not None)` then `probs` will be the original `probs` param
 3. If `probs is None` and `self.probs not None` then `probs` will be `self.probs` with both approaches

In terms of whether the param makes sense to include, I generally like to go by this rule: if I don't have anything actually needing to use it today then I'll leave it out until I need it.

Reply:
The problem is when both are not `None`.
E.g you evaluate and you have self.probs but you want to use that function after you have done else with that self.probs but outside the evaluator.

Reply:
I am moving them to metrics package, conserving the call only for the self attributes of the evaluator.